specifyweb/frontend/js_src/lib/components/DataModel/__tests__/businessRules.test.ts

```diff
@@ -117,10 +117,14 @@ describe('Collection Object business rules', () => {
       resource_uri: collectionObjectUrl,
     });
 
-  beforeEach(() => {
+  beforeAll(() => {
     overwriteReadOnly(schema, 'embeddedCollectingEvent', true);
   });
 
+  afterAll(() => {
+    overwriteReadOnly(schema, 'embeddedCollectingEvent', false);
```

Comment:
can you check if jest would let you put

```ts
const originalEmbeddedCollectingEvent = schema.embeddedCollectingEvent
```

right before the `beforeAll(() => {` line and use that variable instead of hardcoding `false`?

if not, then this is ... fine

Reply:
Fixed in 9a4ded6572ff7e1603638f6c61fabda74fb91daf